Weaken username regexp restriction. Usernames can include more characters than
just letters (e.g. _ or [0-9]).

Author: Jan Synacek <jsynacek@redhat.com>
Reported-By: Robert Scheck
RH-Bugzilla: #894272

--- logwatch-svn127-dist/scripts/services/secure	2013-01-11 10:38:15.778804473 +0100
+++ logwatch-svn127/scripts/services/secure	2013-01-11 10:49:13.381323119 +0100
@@ -247,7 +247,7 @@ while (defined($ThisLine = <STDIN>)) {
       ( $ThisLine =~ /userhelper\[\d+\]: running '([^ ]+)' with [^ ]+ context/) or
       ( $ThisLine =~ /pam_timestamp\(.*:session\): updated timestamp file `\/var\/run\/sudo.*'/) or
       ( $ThisLine =~ /[^ ]*: pam_keyinit(.*:.*): Unable to change GID to [0-9]* temporarily/) or
-      ( $ThisLine =~ /password check failed for user \([a-zA-Z]*\)/) or
+      ( $ThisLine =~ /password check failed for user \(\S*\)/) or
       ( $ThisLine =~ /PAM pam_set_item: attempt to set conv\(\) to NULL/) or
       ( $ThisLine =~ /PAM pam_get_item: nowhere to place requested item/) or
       ( $ThisLine =~ /pam_succeed_if\(.*:.*\): error retrieving information about user [a-zA-Z]*/ ) or
